experiment/references.md

```diff
@@ -8,10 +8,10 @@
 
 1. [http://orgchem.colorado.edu/hndbksupport/specttutor/irchart.html](http://orgchem.colorado.edu/hndbksupport/specttutor/irchart.html)
```

Comment:
Please do not use any http links. Also provide a meaningful short description as the link text instead of using the full link as link text.
Also make sure all links work. If they don't please comment them out and add a comment that they have been commented out because they do not work.
We have discussed this issue at length in the DASS slack channel. I am not sure why it is still not addressed